metabox/README.md

```diff
@@ -150,10 +150,44 @@ configuration = {
 **Note:** Metabox is always going to check **all possible combinations** of
 `releases`, that means that this example will execute 9 test runs.
 
+# Running Metabox with pytest
```

Comment:
I'm wondering if this is a bonus to the `metabox` command, or if it should be promoted as the main way of running Metabox...